### ISSUE TITLE
Add support of the "terraform import" command

### ISSUE THREAD

**Issue by wojcik92michal** [cloudsigma/terraform-provider-cloudsigma#31]
# Feature request

### Description
It would be useful to add a possibility to import existing infrastructure with `terraform import` command (https://www.terraform.io/docs/cli/commands/import.html)

### New or Affected Resource(s)

I guess it would be the best to apply such a change to the all resources.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by pavel-github** [comment]
@wojcik92michal , yes, it's on our roadmap. Let's use this ticket to track the public process here.
